hibernate-core/src/main/java/org/hibernate/engine/jdbc/env/internal/ExtractedDatabaseMetaDataImpl.java

```diff
@@ -181,7 +181,8 @@ public Builder apply(DatabaseMetaData databaseMetaData) throws SQLException {
 			connectionCatalogName = databaseMetaData.getConnection().getCatalog();
 			// NOTE : databaseMetaData.getConnection().getSchema() would require java 1.7 as baseline
 			supportsRefCursors = StandardRefCursorSupport.supportsRefCursors( databaseMetaData );
-			supportsNamedParameters = databaseMetaData.supportsNamedParameters();
+			supportsNamedParameters = jdbcEnvironment.getDialect()
+					.supportsNamedParameters() && databaseMetaData.supportsNamedParameters();
```

Comment:
Hi @gbadner ,
not sure what do you mean with "Overriding from the dialect should probably happen in JdbcEnvironmentImpl"